### PR TITLE
Update some dependencies.

### PR DESCRIPTION
**Pull request by ahmedcharles**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ import typescript from "rollup-plugin-typescript2";
 import screeps from "rollup-plugin-screeps";
 
 let cfg;
-const i = process.argv.indexOf("--dest") + 1;
-if (i == 0) {
+const dest = process.env.DEST;
+if (!dest) {
```
Comment (dmarcuse) [inline rollup.config.js:11]: _incoherent grumbling about type coercion_

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "es2015",
```
Comment (dmarcuse) [inline tsconfig.json:3]: Why is this necessary?

**Review by dmarcuse — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (ahmedcharles) [reply tsconfig.json:3]: It errors if you don't. There was a change to `rollup-plugin-typescript2` which adds the error.
